app/view/main/Map.js

```diff
@@ -54,6 +56,14 @@ Ext.define('CpsiMapview.view.main.Map', {
             listeners: {
                 afterrender: 'initializeMeasureBtn'
             }
+        }, {
+            xtype: 'basigx-button-history',
+            direction: 'BACK',
```

Comment:
Case-sensitive?

Reply:
Yep, is more or less treated like a constant in the BasiGX class. Maybe I'll introduce a static member later, which can be used for this (instead of typing 'BACK' / 'FORWARD').